client/extensions/wp-super-cache/cache-stats.jsx

```diff
@@ -0,0 +1,77 @@
+/**
+ * External dependencies
+ */
+import React, { Component, PropTypes } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import FoldableCard from 'components/foldable-card';
+
+class CacheStats extends Component {
+	static propTypes = {
+		files: PropTypes.array,
+		header: PropTypes.string,
+		translate: PropTypes.func.isRequired,
+	};
+
+	static defaultProps = {
+		files: [],
+	};
+
+	getAge = ( { lower_age, upper_age } ) => {
```

Comment:
Technically this is a method since you call it using `this.getAge` and it's defined inside of a class, but it's also still just a function. But...semantics.

Having said that, I took another look at this code and it looks like it can be moved outside of the React component since it doesn't act on the object at all. Perhaps that's what you were getting at?